SoftLayer/CLI/user/vpn_subnet.py

```diff
@@ -0,0 +1,30 @@
+"""List Users."""
```

Comment:
Change docblock to reflect what this command does.

Reply:
fixed